cosmoz-tabbable-behavior.html

```diff
@@ -171,19 +192,27 @@
 		 * @fires 'tab-select'
 		 * @return {void}        description
 		 */
-		_onIronSelect(e, detail) {
-			var item = detail.item,
+		_onIronSelect(e) {
+			const item = e.detail.item,
 				index = this.items.indexOf(item),
-				opts = {node: item, bubbles: false };
+				opts = {
+					bubbles: false,
+					composed: true,
+					detail: {}
+				};
 
 			if (index > -1) {
-				this.debounce('select-resize', this.notifyResize);
+				this._debouncer = Polymer.Debouncer.debounce(this._debouncer,
+					Polymer.Async.microTask,
+					this.notifyResize
+				);
+				Polymer.enqueueDebouncer(this._debouncer);
 
 				if (!item.isActive) {
-					this.fire('tab-first-select', {}, opts);
+					item.dispatchEvent(new CustomEvent('tab-first-select', opts));
```

Comment:
@Wurper This was the reason why the tests were failing. The event must be dispatched on the item that gets selected.

Reply:
Ah! Thanks!